Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -27,38 +27,37 @@ public class DependencyCollectorDiagnosticListener : IObserver<DiagnosticListene
         /// <returns>
         /// An IDisposable that can be disposed to disable the DependencyCollectorDiagnosticListener.
         /// </returns>
-        public static IDisposable Enable(TelemetryConfiguration configuration = null)
+        public static IDisposable Enable(TelemetryConfiguration configuration = null, IEnumerable<string> excludedDomains = null)
```

Comment:
After discussing offline we'll create the instance of the DependencyTrackingTelemetryModule which, when compiled for .NET Core, will create the DependencyCollectorDiagnosticListener instance.  Then we will set the excluded domains collection, and then when we call Initialize on the module it can call Initialize (Enable) on the listener and pass the excluded domains list on to it.
I'll expose a config option in the .NET Core SDK for customers to provide this list and they can always lookup the listener in the services list later and add to that list.